### PR TITLE
docs: add webhook usage docs in a project

### PR DESCRIPTION
**Pull request by abeizn**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have `npm run build` and `npm run serve` locally before submitting this PR
- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation

# Summary
add webhook usage docs in a project

### Does this close any open issues?
related to https://github.com/apache/incubator-devlake/issues/4462

### Screenshots
image
image

image
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: versioned_docs/version-v0.15/UserManuals/DORA.md [hunk 1]
```diff
@@ -64,113 +66,123 @@ Calculating DORA metrics requires three key entities: **changes**, **deployments
...
+7. Go to the 'Data Connections' page. Create a webhook.
+
+![webhook-add-data-connections](/img/ConfigUI/webhook-add-data-connections.png)
 
-8. Click "Add Incoming Webhook", give it a name, and click "Generate POST URL". DevLake will generate URLs that you can send JSON payloads to push `deployments` and `incidents` to Devlake. Copy the `Deployment` curl command.
-![](https://i.imgur.com/jq6lzg1.png)
-![](https://i.imgur.com/jBMQnjt.png)
+We recommend that you give your webhook connection a unique name so that you can identify and manage where you have used it later.
+
+8. Create a Project first, choose Incoming Webhooks, then you can `Add a Webhook` or `Select Existing Webhooks`. And click "Generate POST URL". DevLake will generate URLs that you can send JSON payloads to push `deployments` and `incidents` to Devlake. Copy the `Deployment` curl command.
```
Comment (likyh) [inline versioned_docs/version-v0.15/UserManuals/DORA.md:104]: `Add a Webhook` here so that step 7 is not required?

**Review by likyh — DISMISSED** [review]

File: versioned_docs/version-v0.15/UserManuals/DORA.md (hunk 1 above)
Comment (abeizn) [reply versioned_docs/version-v0.15/UserManuals/DORA.md:104]: yes, I think the user can add a webhook in project directly

File: docs/Configuration/webhook.md [hunk 2]
```diff
@@ -14,11 +14,17 @@ Go to the 'Data Connections' page. Create a webhook.
 
 We recommend that you give your webhook connection a unique name so that you can identify and manage where you have used it later.
 
-### Step 2 - Use Webhooks
+### Step 2 - Create webhooks connection
 
 Click on Generate POST URL, and you will find four webhook URLs. Copy the ones that suit your usage into your CI or issue-tracking systems. You can always come back to the webhook page to copy the URLs later on.
```
Comment (Startrekzky) [inline docs/Configuration/webhook.md]: Click on `Generate POST URL`, and you will find three webhook URLs.

File: docs/DORA.md [hunk 3]
```diff
@@ -5,47 +5,49 @@ description: >
...
 - **Report**: DevLake provides a built-in DORA dashboard. See an example screenshot below or check out our [live demo](https://grafana-lake.demo.devlake.io/grafana/d/qNo8_0M4z/dora?orgId=1).
-![DORA Dashboard](https://i.imgur.com/y1pUIsk.png)
+  ![DORA Dashboard](https://i.imgur.com/y1pUIsk.png)
 
 DevLake now supports Jenkins, GitHub Action and GitLabCI as data sources for `deployments` data; Jira, GitHub issues, and TAPD as the sources for `incidents` data; Github PRs, GitLab MRs as the sources for `changes` data.
 
-If your CI/CD tools are not listed on the [Supported Data Sources](https://devlake.apache.org/docs/SupportedDataSources) page, have no fear! DevLake provides incoming webhooks to push your `deployments` data to DevLake. The webhook configuration doc can be found [here](https://devlake.apache.org/docs/Configuration/webhook/).
-
+If your CI/CD tools are not listed on the [Supported Data Sources](https://devlake.apache.org/docs/next/Overview/SupportedDataSources) page, have no fear! DevLake provides incoming webhooks to push your `deployments` data to DevLake. The webhook configuration doc can be found [here](https://devlake.apache.org/docs/Configuration/webhook/).
```
Comment (Startrekzky) [inline docs/DORA.md]: You can't use absolute URL of the next version of docs. You should remove the `/next/` part.

The solution is to use relative URL, then Docusaurus won't throw error and the URL will not break after the doc freeze.

File: docs/DORA.md [hunk 4]
```diff
@@ -64,123 +66,132 @@ Calculating DORA metrics requires three key entities: **changes**, **deployments
 In the next section, we'll demonstrate how to configure DevLake to implement DORA metrics for the aforementioned example team.
 
 ### Collect GitHub & Jira data via `blueprint`
+
 1. Visit the config-ui at `http://localhost:4000`
-2. Create a `blueprint`, let's name it "Blueprint for DORA", add a Jira and a GitHub connection. Click `Next Step`
-![](https://i.imgur.com/lpPRZ6v.png)
+2. Create a project: `project1`. And then create a `blueprint`, let's name it "Blueprint for DORA", add a Jira and a GitHub connection. Click `Next Step`
```
Comment (Startrekzky) [inline docs/DORA.md]: Create a project: `project1`. Go to `project1` and create a `blueprint`...

File: versioned_docs/version-v0.15/UserManuals/ConfigUI/webhook.md [hunk 5]
```diff
@@ -14,11 +14,17 @@ Go to the 'Data Connections' page. Create a webhook.
 
 We recommend that you give your webhook connection a unique name so that you can identify and manage where you have used it later.
 
-### Step 2 - Use Webhooks
+### Step 2 - Create webhooks connection
 
 Click on Generate POST URL, and you will find four webhook URLs. Copy the ones that suit your usage into your CI or issue-tracking systems. You can always come back to the webhook page to copy the URLs later on.
```
Comment (Startrekzky) [inline versioned_docs/version-v0.15/UserManuals/ConfigUI/webhook.md]: Click on `Generate POST URL`, and you will find three webhook URLs.

File: versioned_docs/version-v0.15/UserManuals/DORA.md [hunk 6]
```diff
@@ -64,113 +66,123 @@ Calculating DORA metrics requires three key entities: **changes**, **deployments
...
 For more details, please refer to our [blueprint manuals](https://devlake.apache.org/docs/UserManuals/ConfigUI/Tutorial).
 
 ### Collect CircleCI data via `webhook`
 
 Using CircleCI as an example, we demonstrate how to actively push data to DevLake using the Webhook approach, in cases where DevLake doesn't have a plugin specific to that tool to pull data from your data source.
 
-7. Visit "Data Connections" page in config-ui and select "Issue/Deployment Incoming Webhook".
+7. Go to the 'Data Connections' page. Create a webhook.
+
+![webhook-add-data-connections](/img/ConfigUI/webhook-add-data-connections.png)
```
Comment (Startrekzky) [inline versioned_docs/version-v0.15/UserManuals/DORA.md]: Add indent?

File: versioned_docs/version-v0.15/UserManuals/DORA.md [hunk 7]
```diff
@@ -64,113 +66,123 @@ Calculating DORA metrics requires three key entities: **changes**, **deployments
 In the next section, we'll demonstrate how to configure DevLake to implement DORA metrics for the aforementioned example team.
 
 ### Collect GitHub & Jira data via `blueprint`
+
 1. Visit the config-ui at `http://localhost:4000`
-2. Create a `blueprint`, let's name it "Blueprint for DORA", add a Jira and a GitHub connection. Click `Next Step`
-![](https://i.imgur.com/lpPRZ6v.png)
+2. Create a project: `project1`. And then create a `blueprint`, let's name it "Blueprint for DORA", add a Jira and a GitHub connection. Click `Next Step`
```
Comment (Startrekzky) [inline versioned_docs/version-v0.15/UserManuals/DORA.md]: Text change:
Create a project: `project1`. Go to `project1` and create a `blueprint`...

**Review by Startrekzky — CHANGES_REQUESTED** [review]

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**